Scripts/Core/EditorVR.Viewer.cs

```diff
@@ -37,6 +37,8 @@ class Preferences
 
 			const float k_CameraRigTransitionTime = 0.25f;
 
+            static Collider[] s_CachedColliders = new Collider[20];
```

Comment:
Const the "20." Also, looking at the context here, I don't think more than 2 are necessary. We're specifically looking for the player body triggers, one directly above the head and one slightly behind and above the head.

Reply:
Actually, never mind. If there are a bunch of colliders near your head, that would fail. Something in me still wants this to be a power of two...

Reply:
I'm down with 32